File: connectionSystem/keygen/keygen.go
package keygen

import (
	"math/rand"
)

func GenerateKey(len int) string {

	outStr := ""

	for i := 0; i < len; i++ {
		randASCII := string(randNumber(65, 90))
		outStr = outStr + randASCII
	}

	return outStr

}

func randNumber(min, max int) int {
	return min + rand.Intn(max-min)
}

File: pumpapp/pumpapp/src/classes/socket/keygen.ts



class Keygen{

    public static generateKey(len:number):string
    {
        
        let outStr:string = "";
        
        
        for(let i = 0; i < len; i++){
            let asciiInt:number = Math.floor(Math.random() * (90-65) + 65);   
            outStr += String.fromCharCode(asciiInt)
        }

        return outStr;
    
    
    }

}



export default Keygen